magenta-lib/src/main/scala/magenta/deployment_type/S3.scala

```diff
@@ -34,7 +34,11 @@ object S3 extends DeploymentType {
   )
 
   val publicReadAcl = Param[Boolean]("publicReadAcl",
-    "Whether the uploaded artifacts should be given the PublicRead Canned ACL. (Default is true!)"
+    """
+      |Whether the uploaded artifacts should be given the PublicRead Canned ACL. (Default is true!) Note that the
+      |default bucket setting is to block all public access, which doesn't play nicely with this parameter when set
```

Comment:
I'd mention AWS here. We're using the word default to mean two different things (default in Riff Raff, default for S3 buckets in AWS). We're in RiffRaff land so we can help the reader by signposting the latter.

Reply:
Great shout, I'll update this 👍

Reply:
(Hopefully) improved/fixed this in 1bcc2df9e03d66493198c26dac8809e177f803c6.